HTML/HTML.sublime-syntax

```diff
@@ -164,7 +164,7 @@ contexts:
         - include: script-javascript-content
 
   script-javascript-content:
-    - match: (?=\S)
```

Comment:
This change fixes https://forum.sublimetext.com/t/st4-cant-toggle-comment-block-on-script-tags-in-html-files/57723

Reply:
... needs to be ported to ASP/JSP as well.

Reply:
... but it breaks html comments support

![grafik](https://user-images.githubusercontent.com/16542113/119331962-3e327380-bc88-11eb-97b6-0d9d7e3e8d9b.png)

Reply:
To be fixed via #2831

Reply:
That looks good. I would like to rebase this PR on that once it is merged.